Tests/GRDBTests/FTS5TableBuilderTests.swift

```diff
@@ -1,6 +1,9 @@
 #if SQLITE_ENABLE_FTS5
 import XCTest
 import GRDB
+#if GRDBCIPHER
+import SQLCipher
+#endif
```

Comment:
To be removed if we replace `sqlite3_libversion_number()` with `Database.sqliteLibVersionNumber`.